ssl/ssl_lib.c

```diff
@@ -5582,3 +5582,15 @@ void SSL_set_allow_early_data_cb(SSL *s,
     s->allow_early_data_cb = cb;
     s->allow_early_data_cb_data = arg;
 }
+
+#ifdef SSL_DEBUG
+/* 
+ * Debug routine, uses the prefix with a possible int argument |ival|
+ */
+void ssl_debug_data_dump(const char* prefix, unsigned int ival, void *data, size_t len)
```

Comment:
Suggest making this: `void ssl_debug_data_dump(const char *prefix, void *data, size_t len, ...)`
Add `__attribute__ ((format (printf, 1, 4)))` on platforms where this is available.  Essentially, don't force a single integer argument which might or might not be used.

Reply:
Since this was wrapped with a macro, I was avoiding using varargs. But I suppose it depends on the CPP.